src/main/java/org/prebid/server/bidder/adhese/AdheseBidder.java

```diff
@@ -57,6 +58,7 @@
     private static final String ORIGIN = "JERLICIA";
     private static final String QUERY_PARAMETER_GDPR = "/xt";
     private static final String QUERY_PARAMETER_REFERER = "/xf";
+    private static final String QUERY_PARAMETER_IFA = "/xz";
```

Comment:
Minor. Usually most valuable thing can placed at first position, like `IFA_QUERY_PARAMETER`.
If you're ok with it, pls check for other fields.